networking_nsxv3/common/constants.py

```diff
@@ -18,6 +18,14 @@
 NSXV3_REVISION_SCOPE = "revision_number"
 NSXV3_AGE_SCOPE = "age"
 
+# Start section: SELECTIVE LOGGING
+NSXV3_AGENT_SCOPE = "agent_id"
```

Comment:
That "agent_id" tag was removed intentionally. Do we need it for the Selective Logging really?